tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+# a single instance of ``pyplot``. This convention uses ``plt.subplots()``
+# and defaults to one Figure, ``fig``, and one Axes, ``ax``. The
+# `Configuration`_ section below contains additional information about
+# manipulating visuals, multiple visulizations, and other modifications.
+#
+# Using the OOP approach allows for ``fig`` and ``ax`` to use separate methods
+# to manipulate the visualization. Instead of using the module ``pyplot`` for
+# all instances of managing objects, the specfic Axes refers to OOP usage and
+# manages the respective data.
+#
+# Implicit: ``pyplot``
+# --------------------
+#
+# Implicit programming for Matplotlib centers around using the ``pyplot``
+# module. The Figure and Axes are automatically generated by the module.
+# The methods and functions within the module take incoming data as arguments.
+# Additional parts of the Figure are also available through the module
+# methods.
+
+# Implicit programming with pyplot
+
+y4 = chk_acct_10
+y5 = svg_acct_10
+# Assigning former data to labeled variable.
+# Previous variables are still referenced.
+
+plt.plot(x, y1, label='Income')
+plt.plot(x, y4, label='Checking Account')
+plt.plot(x, y5, label='Savings Account')
+# Module plots multiple data points on implicitly generated Axes.
+
+plt.xlabel('Month')
+plt.ylabel('USD')
+plt.title("Personal Financial Tracking from 2010")
+plt.legend()
+# Module methods generate parts of Figure.
+
+plt.show()
+# The module displays the Figure.
+
+##############################################################################
+#
+# In the example above, the ``pyplot`` module contains its own methods of
+# actionable tasks for the data. The ``plt.plot`` plots data as a line graph
+# with various keyword arguments as customizable options. The module also
+# includes other methods for generating parts of the visualization. These parts
+# use different methods from the OOP approach.
+#
+# .. note::
+#
+#    The names and spelling for methods may be similar for both explicit and
+#    implicit approaches. Errors may occur when using the wrong corresponding
+#    method. Confirm with the documentation API of `~.axes.Axes` for explicit
+#    and :mod:`matplotlib.pyplot` for implicit or other respective method
+#    names.
+#
+# Configuration
+# =============
+#
+# There are two main parts to building a visualization with Matplotlib, the
+# ``Figure`` and the ``Axes``.
+#
+# Components of Matplotlib Figure
+# -------------------------------
+#
+# The image below depicts each visible element of a Matplotlib graph. The
+# graphic uses Matplotlib to display and highlight each individual part of the
+# visualization. To see how the programming operates, the source code is
+# available at :ref:`sphx_glr_gallery_showcase_anatomy.py`.
+#
+# .. note::
+#
+#     ``Figure`` and ``Axes`` identify empty regions of the diagram;
+#     however, these elements are foundational in operation.
```

Comment:
might be useful to show a small empty figure here to identify?

Reply:
Agreed!